furbot.rb

```diff
@@ -197,6 +197,58 @@
     msg.channel.send("https://duckduckgo.com/?q=#{CGI.escape(term)}")
   end
 
+  on :message, /!log (\A[a-z|A-Z|\d|.|_|-|#|,]{1,20}\z)$/ do |msg, logsearchtext1|
+    # Showing first newest irclog search result
+    logresultcounter1 = 0
+    logfilelist1 = Dir['/logs/plainlogs/*.log'].sort.reverse
+    logfilelist1.each do |logfilename1|
+       File.readlines(logfilename1).reverse_each do |filelogline1|
+          completelogline1 = "#{logfilename1[-14..-5]} #{filelogline1}"
+          if (completelogline1[logsearchtext1])
+             logresultcounter1 = logresultcounter1 + 1
+             msg.channel.send("#{completelogline1}")
+             exit
```

Comment:
A call to `#exit` will terminate the program (i.e. make furbot quit). I don’t think this is a good idea to use.